src/content/docs/guides/json/0-json_intro.mdx

```diff
@@ -1,142 +1,192 @@
+
+### Overview of JSON in SplashKit
+
+SplashKit simplifies the process of working with JSON files in your games. Functions are provided for reading JSON files, so that we can easily read values and load configurations or game data. Additional functions are provided for writing JSON files, so that we can save configurations and game data.
+
+### Getting Started with JSON in SplashKit
+
+To begin using JSON in SplashKit, we must have our files in the correct locations. Run the following command in your project directory to generate the resources folder.
+
+```shell
+skm resources
+```
+
+This command creates sub-folders for each type of resource. One of these is named `json` and that is where we place our JSON files. To begin lets take the example JSON file above and place it into the `json` folder with the name `game_data.json`. To access the values in this file we can now do the following:
+
+<Tabs syncKey="code-language">
+  <TabItem label="C++">
+
+  ```cpp
+  #include "splashkit.h"
+  int main()
+  {
+      json game_data;
+      game_data = json_from_file("game_data.json");
+      string game_title = json_read_string(game_data, "gameTitle");
+
+      write("Game Title: ");
+      write_line(game_title);
+
+      free_json(game_data);
+
+      return 0;
+  }
+  ```
+
+  </TabItem>
+  <TabItem label="Python">
+
+```python
+from splashkit import *
+
+def main():
+    
+    game_data = json_from_file("game_data.json")
+    game_title = json_read_string(game_data, "gameTitle")
+    
+    if not game_title:
+        print("Error retrieving gameTitle.")
+    else:
+        print("Game Title:", game_title)
+    
+     free_json(game_data)
+   
+
+if __name__ == "__main__":
+    main()
+    ```
+
+   </TabItem>
+
+  
+</Tabs>
+
+In this code example, we're first using [Json From File](/api/json/#json-from-file) to load a JSON object with the details from the `game_data.json` file.
+Then we are getting the value of the key that matches `gameTitle` using [Json Read String](/api/json/#json-read-string). We write this to the console and free the JSON object using [Free Json](/api/json/#free-json) as we exit. By freeing the JSON object we are deallocating any memory that has been assigned to it, preventing any memory-related errors such as a `Segmentation Fault`. We can build this program with the following command.
+
+<Tabs syncKey="code-language">
+  <TabItem label="C++">
+
+  ```shell
+  g++ program.cpp -l SplashKit -o json_program
+  ```
+
+  </TabItem>
+</Tabs>
+
+And run it with:
+
+<Tabs syncKey="code-language">
+  <TabItem label="C++">
+
+  ```shell
+  ./json_program
+  ```
+  
+  </TabItem>
+  <TabItem label="Python">
+
+  ```shell
+  skm python3 program.py
+  ```
+</TabItem>
+</Tabs>

```

Comment:
This section doesn't display well. Doing the following would look far nicer:

In the C++ tab section:

  ```shell
  # Compile the program
  g++ program.cpp -l SplashKit -o json_program

  # Run the program
  ./json_program
  ```

Then in the python tab:
  ```shell
  skm python3 program.py
  ```
  
 So instead of doing two lots, combine into one.